src/Helper.ts

```diff
@@ -304,14 +318,37 @@ export class Helper {
       manifestGroup.addMetadata(manifestMetadata, true);
     }
 
-    if (this.manifest.getDescription().length) {
-      const metadataItem: LabelValuePair = new LabelValuePair(locale);
-      metadataItem.label = new PropertyValue([
-        new LocalizedValue("description", locale),
-      ]);
-      metadataItem.value = this.manifest.getDescription();
-      (<IMetadataItem>metadataItem).isRootLevel = true;
-      manifestGroup.addItem(<IMetadataItem>metadataItem);
+    //include summary if no description in metadata
+    if (this.manifest.getSummary().length) {
+      let descriptionMatches = false;
+      let description = "";
+      for (let i = 0; i < manifestMetadata.length; i++) {
+        const item = manifestMetadata[i];
+        let labelText = "";
+        labelText = String((item as any).getLabel() || "").trim();
+        if (labelText.toLowerCase() === "description") {
+          description = String((item as any).getValue(locale) || "").trim();
+          break;
+        }
+      }
+      const summaryText = String(
+        (this.manifest.getSummary() as any).getValue(locale) || ""
+      ).trim();
+      if (summaryText === description) {
+        descriptionMatches = true;
+      }
```

Comment:
As written, this is only comparing things when the label is "description," but I don't think it's safe to make any assumptions about what labels will be used. I'd just compare every value; something like this:


```suggestion
    const summaryText = String(
      (this.manifest.getSummary() as any).getValue(locale) || ""
    ).trim();
    if (summaryText.length > 0) {
      let descriptionMatches = false;
      let description = "";
      for (let i = 0; i < manifestMetadata.length; i++) {
        const item = manifestMetadata[i];
        description = String((item as any).getValue(locale) || "").trim();
        if (description.length > 0 && summaryText === description) {
          descriptionMatches = true;
          break;
        }
      }
```

(I did a little other rearranging here in an effort to improve the consistency of the matching. I haven't tested this code, though, so apologies if I've made any mistakes!)